src/traffic_server/traffic_server.cc

```diff
@@ -1813,10 +1813,10 @@ main(int /* argc ATS_UNUSED */, const char **argv)
   quic_NetProcessor.init();
 #endif
 
-  // If num_accept_threads == 0, let the ET_NET threads to set the condition variable,
+  // If num_accept_threads == 0, let the ET_NET threads set the condition variable,
   // Else we set it here so when checking the condition variable later it returns immediately.
-  if (num_accept_threads == 0) {
-    eventProcessor.schedule_spawn(&init_HttpProxyServer, ET_NET);
+  if (num_accept_threads == 0 || command_flag) {
```

Comment:
Shouldn't that be `! command_flag`?

Reply:
no, we need this to wait when `command_flag` is true, new Bryan found a problem and this fixes it.

Reply:
We should really not start up the threads at all, but I suppose that's OTSOTPR.

Reply:
not familiar with the command mode, maybe I'll come bug you to teach me about it :D